tutorials/dataframe/distrdf001_vecOpsHEP.py

```diff
@@ -0,0 +1,89 @@
+## \file
+## \ingroup tutorial_dataframe
+## \notebook -draw
+## Use RVecs to plot the transverse momentum of selected particles.
+##
+## This tutorial is based on tutorial `df017_vecOpsHEP` and also shows how to
+## use the experimental `ROOT.RDF.Experimental.Distributed` package to run the
+## workflow distributedly through [Apache Spark](http://spark.apache.org/).
+##
+## \macro_code
+## \macro_image
+##
+## \date March 2021
+## \author Vincenzo Eduardo Padulano
+## \date March 2018
+## \authors Danilo Piparo (CERN), Andre Vieira Silva
+
+import os
+import sys
+
+import ROOT
+
+filename = ROOT.gROOT.GetTutorialDir().Data() + "/dataframe/df017_vecOpsHEP.root"
+treename = "myDataset"
+
+# Synchronize PYSPARK_PYTHON variable with the current Python executable.
+# Needed to avoid mismatch between Python versions in the process running this
+# script and in the Spark executors on the machine running the test. For more
+# info see the corresponding configuration parameter at
+# https://spark.apache.org/docs/latest/configuration.html#runtime-environment
+os.environ["PYSPARK_PYTHON"] = sys.executable
+
+
+def WithPyROOT(filename):
+    """
+    Fill an histogram with transverse momentum of selected particles. The
+    selection and filling is performed in a for loop.
+    """
+    from math import sqrt
+    f = ROOT.TFile(filename)
+    h = ROOT.TH1F("pt", "With PyROOT", 16, 0, 4)
+    for event in f.myDataset:
+        for E, px, py in zip(event.E, event.px, event.py):
+            if E > 100:
+                h.Fill(sqrt(px*px + py*py))
+    h.DrawCopy()
+
+
+def WithRDataFrameVecOpsJit(treename, filename):
+    """
+    Fill an histogram with transverse momentum of selected particles. The
+    selection and filling is handled by RDataFrame using RVec.
+    """
+    f = ROOT.RDataFrame(treename, filename)
+    h = f.Define("good_pt", "sqrt(px*px + py*py)[E>100]")\
+         .Histo1D(("pt", "With RDataFrame and RVec", 16, 0, 4), "good_pt")
+    h.DrawCopy()
```

Comment:
let's keep tutorials focused -- you can just point to the other tutorials rather than including non-distributed ways to do the same thing here

Reply:
I imagined this would be the point of this tutorial, but if you think it's not worth it then I would remove it altogether and make another one that doesn't reference `df017`

Reply:
Tutorial has been deleted in the latest commit